gcn_email/core.py

```diff
@@ -99,7 +99,7 @@ def kafka_message_to_email(message):
     email_message = EmailMessage()
     if topic.startswith("gcn.classic.text."):
         email_message.set_content(message.value().decode())
-    elif topic.startswith("gcn.classic.voevent."):
+    elif ".voevent." in topic:
```

Comment:
Please add a comment to explain why the match is not `topic.startswith('gcn.classic.voevent.')`.